### PR TITLE
[feature] Add an add on manager dashboard

### PR DESCRIPTION
**Pull request by wanjikum**
## JIRA TICKET NAME:
[OWA-46](https://issues.openmrs.org/browse/OWA-46) Create a dashboard for the add-on manager

### SUMMARY:
- Edit the webpack config file to bundle the bootstrap from the libs folder and also images
- It's should have a header that has some the information about the user like his location and his role and also logout button.
- The dashboard should have a icons that will lead you to the manage application page and the manage settings page.
- Write tests for the dashboard.
- Ignore static files extensions from being tested
- Eliminate upgrade testing libraries to eliminate 'ReactTestUtils has been moved to react-dom/test-utils and  Shallow renderer has been moved to react-test-renderer/shallow.' warnings.

### CODE REVIEW

File: app/css/openmrs-addonmanager.css [hunk 1]
```diff
@@ -70,3 +70,125 @@ header .logo img {
...
+    color: purple;
+}
+
+.MainHomePage h2{
+    font-size: 1.3em;
+    color: #009384;
+    margin-left: 5px;
+}
+
+.HomePageBody{
```
Comment (dkayiwa) [inline app/css/openmrs-addonmanager.css]: For some, you start with lower case, for others, upper. Do you do this intentionally? 😊

File: app/js/components/common/header.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+import React, {Component} from 'react';
+import {Link, IndexLink} from 'react-router';
+import {ApiHelper} from '../../helpers/apiHelper';
+
```
Comment (dkayiwa) [inline app/js/components/common/header.js:14]: Do you think we can easily create some sort of reusable component that even other andela apprentice groups can just reuse for this header?

File: app/js/components/common/header.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+      this.setState({currentLogOutUrl: customUrl});
+      return customUrl;
+        
+    });
+  }
+
+  fetchLocation(url) {
+    const apiHelper = new ApiHelper(null);
+    const getData = new Promise(function(resolve, reject) {
+      apiHelper.get(url).then(response => {
```
Comment (dkayiwa) [inline app/js/components/common/header.js:63]: Are other groups aware of these utility classes like apiHelper?

File: app/css/openmrs-addonmanager.css (hunk 1 above)
Comment (wanjikum) [reply app/css/openmrs-addonmanager.css]: Okay, thank you for that. Consistency is important. I am changing this now.

File: app/js/components/common/header.js (hunk 2 above)
Comment (wanjikum) [reply app/js/components/common/header.js:14]: Yes we can.

File: app/js/components/common/header.js (hunk 3 above)
Comment (wanjikum) [reply app/js/components/common/header.js:63]: Yes, they are. The August group is using [this](https://github.com/openmrs/openmrs-owa-obsadmin/blob/master/app/js/utilities/apiHelper.js).

Comment (dkayiwa) [reply app/js/components/common/header.js:63]: Awesome. Please share as many goodies as you can. 😊

**Pull request merged**